.github/workflows/ci.yml

```diff
@@ -111,7 +111,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.7", "3.8", "3.9"]
```

Comment:
I think we should still support Python 3.7

Reply:
See my comment above to your previous comment about this.